src/constants/roles.ts

```diff
@@ -0,0 +1,17 @@
+export const ROLES = [
```

Comment:
Do you want me to expose a query to get these dynamically based on edge types?

Reply:
Yeah I think that would be beneficial because I would also like to use that when listing the team members.

Reply:
Awesome possum, I'll get on that this evening-ish!